visualizer/data-tree.js

```diff
@@ -48,9 +48,16 @@ class DataTree {
     return false
   }
 
-  sortFramesByHottest () {
-    // Flattened tree, sorted hottest first, excluding the 'all stacks' root node
-    this.flatByHottest = this.getFlattenedSorted(this.getStackTopSorter())
+  sortFramesByHottest (customRootNode) {
```

Comment:
nice approach! i was expecting a `.filter()` call on flatByHottest, but i like this better :+1: